src/plugins/graph/components/graph.tsx

```diff
@@ -29,6 +29,7 @@ import {AttributeType} from "../../../models/data/attribute";
 import {useDataTips} from "../hooks/use-data-tips";
 
 import "./graph.scss";
+import "./graph-clue-styles.scss";
```

Comment:
This and [graph-clue-styles.scss](https://github.com/concord-consortium/collaborative-learning/pull/1696/files#diff-2800e482270c47fb7143b21fb3c104e88557efda09ca0656b0b2592ec0e3e7e7) are very simple style modifications to make the graph look slightly more CLUE-ish.